dataflow/src/ops/trigger.rs

```diff
@@ -0,0 +1,223 @@
+use std::collections::HashMap;
+
+use prelude::*;
+
+use hyper::Client;
+use tokio_core::reactor::Core;
+use serde::Serialize;
+use serde_json;
+use std::thread;
+
+/// Applies the identity operation to the view. Since the identity does nothing,
+/// it is the simplest possible operation. Primary intended as a reference
```

Comment:
I think this comment is a little misleading. The operator does not *just* apply the identify operator, and arguably that's not even it's primary purpose. It'd be better if we here gave a clear description of what this operators is used for.